packages/client-adapters/src/errors/cedict-error.js

```diff
@@ -0,0 +1,16 @@
+export default class CedictError extends Error {
```

Comment:
what do you think about naming this something more generic, like CodedError (or something bettter if you can come up with it)? It's not really specific to Cedict is it? It's just a type of a error that has a code associated with it. We could then reuse the same class in other circumstances.

Reply:
I think that's a great idea, I will think about what to rename it to.

Reply:
@balmas, how about `RemoteError`? Errors in client adapters are classified by what type of error it is, not what format it has. Following this logic, a `RemoteError` will be an error generated by a remote server. Since a `ResponseMessage` now includes an error code if something goes wrong, we can say that a `RemoteError` will always have an error code, even when other services (i.e. non-CEDICT) will be used. 

Does it make sense?

Reply:
Yes I like this idea